griptape/processors/prompt_driver_processor.py

```diff
@@ -0,0 +1,26 @@
+from typing import Callable, Dict, List, Any
+from .base_processors import BasePromptStackProcessor
+import attr
+
+
+@attr.s
```

Comment:
Can you please use `attrs` syntax consistent with the rest of the framework? E.g., `define` for class and `field` for properties.